node/src/chain_spec/turing.rs

```diff
@@ -183,23 +183,73 @@ fn testnet_genesis(
 		vesting: VestingConfig { vesting_schedule },
 		xcmp_handler: XcmpHandlerConfig { chain_data: xcmp_handler_data },
 		asset_registry: AssetRegistryConfig {
-			assets: vec![(
-				0,
-				orml_asset_registry::AssetMetadata::<Balance, CustomMetadata>::encode(
-					&orml_asset_registry::AssetMetadata {
-						decimals: TOKEN_DECIMALS,
-						name: "Native".as_bytes().to_vec(),
-						symbol: TOKEN_SYMBOL.as_bytes().to_vec(),
-						existential_deposit: 100_000_000,
-						location: Some(V1(MultiLocation { parents: 0, interior: Here })),
-						additional: CustomMetadata {
-							fee_per_second: Some(416_000_000_000),
-							conversion_rate: None,
+			assets: vec![
```

Comment:
See https://github.com/OAK-Foundation/OAK-blockchain/pull/319#discussion_r1048844541

The new assets should be moved out from this method.